### PR TITLE
Added code for avatar and badge

### PR DESCRIPTION
**Pull request by lalith96**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link href="./components/css/styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <a href="./components/avatar/index.html">Avatar</a>
+    <a href="./components/alert/index.html">Alert</a>
```
Comment (SaiArunesh) [inline index.html]: please add Link to badge  here

File: components/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  height: 13%;
+  position: absolute;
+  top: 70%;
+  right: 0px;
+  background-color: var(--green-color);
+  border: 2px solid var(--white-color);
+  border-radius: 50%;
+}
+
+.badge-on-icon {
```
Comment (SaiArunesh) [inline components/css/styles.css:87]: badge-on-icon can define the sizing etc. bg-color can be moved to a different css selector. say class="badge-on-icon bg-green" or   class="badge-on-icon online-red"

**Review by SaiArunesh — CHANGES_REQUESTED** [review]
CSS selector naming can be more specific . Please host using vercel so that it is easier to review vercel.app

File: index.html (hunk 1 above)
Comment (lalith96) [reply index.html]: Added link to badge.

File: components/css/styles.css (hunk 2 above)
Comment (lalith96) [reply components/css/styles.css:87]: Added background colours in different classes.

**Comment by lalith96** [comment]
> CSS selector naming can be more specific . Please host using vercel so that it is easier to review vercel.app

**Comment by lalith96** [comment]
Thank you @SaiArunesh for reviewing and providing feedback. Please provide me with some examples for CSS selector naming from my code where I can improve. And I hosted in Netlify https://lalith-component-library.netlify.app .

**Review by ruchiragoswami — APPROVED** [review]
Looks good Lalith :)

**Review by SaiArunesh — APPROVED** [review]

**Pull request merged**